assets/about_us/woman-exercising_old.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="316" height="237" viewBox="0 0 316 237" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If this is an old image, should we just delete it?

Reply:
I left it only b/c the new image wasn't yet rendering as expected.  I've brought it up w Vic, she is going to investigate and perhaps alter the svg somewhat so it renders correctly.  

It's a vexing issue where the svg file when opened on its own in the browser looks fine but in localhost:3000 looks the same w undesired lines on the figure's pants, even after hard refresh and triple checking the code is referring to the right file.